src/main/java/com/hedvig/backoffice/graphql/resolvers/QuestionGroupResolver.kt

```diff
@@ -0,0 +1,34 @@
+package com.hedvig.backoffice.graphql.resolvers
+
+import com.coxautodev.graphql.tools.GraphQLResolver
+import com.hedvig.backoffice.graphql.GraphQLConfiguration
+import com.hedvig.backoffice.graphql.types.Member
+import com.hedvig.backoffice.graphql.types.questions.QuestionGroupType
+import com.hedvig.backoffice.services.members.MemberService
+import com.hedvig.backoffice.services.personnel.PersonnelService
+import graphql.schema.DataFetchingEnvironment
+import org.springframework.stereotype.Component
+
+@Component
+class QuestionGroupResolver
+  (
+  private val memberService: MemberService,
+  private val personnelService: PersonnelService
+) : GraphQLResolver<QuestionGroupType?> {
+  fun getMember(
+    QuestionGroup: QuestionGroupType,
```

Comment:
Camelcase on variables
```suggestion
    questionGroup: QuestionGroupType,
```